src/Compilers/CSharp/Test/Semantic/Semantics/ScriptSemanticsTests.cs

```diff
@@ -717,6 +716,56 @@ public void ERR_VariableUsedBeforeDeclaration()
                 Diagnostic(ErrorCode.ERR_UseDefViolation, "x").WithArguments("x").WithLocation(1, 22));
         }
 
+        [WorkItem(550)]
+        [Fact]
+        public void ERR_VariableUsedBeforeDeclaration_02()
+        {
+            var c = CreateSubmission(
+@"object b = a;
```

Comment:
If we ever want to converge "standard" C# with the interactive variant, we'd have to get LDM signoff on this not being an error.